Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -169,6 +208,14 @@ export function destroy(element, elementId) {
 
     if (instance) {
         instance.destroy();
+
+        if (instance.errorClassWatcher) {
+            instance.errorClassWatcher.disconnect();
+        }
+
+        if (instance.successClassWatcher) {
+            instance.successClassWatcher.disconnect();
+        }
```

Comment:
calling .destroy before doing the other instance cleanup has no negative side effects right?

Reply:
Reverted just in case.